src/plugins/intel_gna/cmake/libGNAConfig.cmake

```diff
@@ -36,7 +36,12 @@ if(libGNA_FIND_REQUIRED_KERNEL)
 endif()
 
 if(libGNA_FIND_REQUIRED_API)
-    find_path(libGNA_INCLUDE_DIRS gna2-api.h PATHS "${GNA_EXT_DIR}/include")
+    if(NOT LINUX_OS_NAME STREQUAL "CHROMIUMOS")
+        find_path(libGNA_INCLUDE_DIRS gna2-api.h PATHS "${GNA_EXT_DIR}/include")
+    else()
+        find_path(libGNA_INCLUDE_DIRS gna2-api.h PATHS ${GNA_INCLUDE_LOC})
```

Comment:
where `GNA_INCLUDE_LOC` is defined?

Reply:
GNA_INCLUDE_LOC will be provided in ebuild file. https://chromium.googlesource.com/chromiumos/docs/+/HEAD/portage/ebuild_faq.md#ebuild-faq